assets/js/base/components/quantity-selector/index.js

```diff
@@ -18,15 +17,16 @@ const QuantitySelector = ( {
 	className,
 	quantity = 1,
 	minimum = 1,
-	maximum = undefined,
+	maximum,
 	onChange = () => null,
 	itemName = '',
 	disabled,
 } ) => {
 	const classes = classNames( 'wc-block-quantity-selector', className );
 
+	const hasMaximum = maximum !== undefined;
```

Comment:
Not blocking, but generally I think it's better to get in the habit of doing typeof checks because _undeclared_ variables will trigger a `undefined` error with this expression.  Usage here is fine in this case because `maximum` is declared.

Reply:
Thanks for the nudge :) 97f65a80cc10e2edc646194d232692b6d67cac87